README.md

```diff
@@ -68,8 +68,8 @@ app.get('/products', (req, res, next) => {
   res.json({msg: req.pagination})
 })
 
-app.listen(3000, () => {
-  console.log('web server listening on port 80')
+const server = app.listen(3000, () => {
```

Comment:
Great catch! Thanks a lot for this PR! @devgar 

Maybe can we use just `port`? I love the idea of using `server.address().port` but not all the developers are used to it. 

```js
const port = 3000;

app.listen(port, () => {
  console.log(`web server listening on port ${port}`)
})
```

What do you think?